src/cartesian/Area.tsx

```diff
@@ -95,10 +106,10 @@ class Area extends PureComponent {
     animationEasing: 'ease',
   };
 
-  static getBaseValue = (props, xAxis, yAxis) => {
+  static getBaseValue = (props: Props, xAxis: Props['xAxis'], yAxis: Props['yAxis']): number => {
```

Comment:
If you agree to create `InternalXAxisProps` and `InternalYAxisProps`, we can change `Props[`xAxis`]` and `Props[`yAxis`]` to them.

Reply:
@sainthkh If we add `InternalXAxisProps` in `utils/types.ts`, `utils/types.ts` need to import `XAxis`, but `XAxis` need to import `utils/types.ts`. It leads to a cycle.

Reply:
What I was talking about is putting them under `/cartesian/types.ts`. Because those `props` are only used in `/cartesian`.

Reply:
Good. Let's leave them as-is for now. We need to think about them later.